sechub-commons-model/src/main/java/com/mercedesbenz/sechub/commons/model/SecHubRemoteCredentialSupport.java

```diff
@@ -0,0 +1,65 @@
+package com.mercedesbenz.sechub.commons.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SecHubRemoteCredentialSupport {
+    private final String PDS_PREPARE_REMOTE_CREDENTIAL_LIST = "PDS_PREPARE_REMOTE_CREDENTIAL_LIST";
+    private static final Logger LOG = LoggerFactory.getLogger(SecHubRemoteCredentialSupport.class);
+
+    public List<SecHubRemoteCredentialContainer> resolveCredentialsForLocation(SecHubRemoteCredentialConfiguration configuration, String location) {
+        return matchRemoteCredentialsFromLocationConfiguration(configuration, location, null);
+    }
+
+    public List<SecHubRemoteCredentialContainer> resolveCredentialsForLocation(SecHubRemoteCredentialConfiguration configuration, String location,
+            String type) {
+        TypeFilter.AcceptTypeFilter acceptTypeFilter = new TypeFilter.AcceptTypeFilter(type);
```

Comment:
You can import the subclass in imports directly and use it without the "TypeFilter." prefix. 
It would be more readable and also easier for refactoring (if the class would be moved to its own location)